content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -1,67 +1,236 @@
+After-Sales services
 ====================
 
-As your business grows, having the right tool to support your helpdesk team on recording, tracking
-and managing issues raised easy and efficiently, is key. Odoo’s Helpdesk application allows you to
-generate credit notes, manage returns, products, repairs, grant coupons, and even plan onsite
-interventions from a ticket’s page.
+*After-Sales* services can be configured in the :guilabel:`Helpdesk` application within an
+individual :guilabel:`Team`. Once enabled, users can issue refunds, process returns, generate
+coupons, and/or schedule repairs and field service interventions directly from a ticket.
 
-Set up the after sales services
+Set up the After-Sales services
 ===============================
 
-Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and enable the after sales
-options: *Refunds, Returns, Coupons, Repairs and Onsite Interventions*.
+Go to :menuselection:`Helpdesk --> Configuration --> Teams` to select a :guilabel:`Team`. Scroll to
+the :guilabel:`After-Sales` section and choose which of the following options to enable:
 
-.. image:: after_sales/aftersales1.png
+- :guilabel:`Refunds`: Issues credit notes to refund a customer, or adjust the remaining amount due.
+- :guilabel:`Coupons`: Offers discounts and free products through an existing coupon program.
+- :guilabel:`Returns`: Initiates a product return from a customer through a reverse transfer.
+- :guilabel:`Repairs`: Creates repair orders for broken or faulty products.
+- :guilabel:`Field Service`: Plans onsite intervention through the Field Service application.
+
+.. image:: after_sales/after-sales-enable.png
+   :align: center
+   :alt: View of a team's settings page with a focus on the after-sales services.
+
+.. warning::
+   As all of the After-Sales services require integration with other applications, enabling any of
+   them may result in the installation of additional modules or applications. *Installing a new
+   application on a One-App-Free database will trigger a 15-day trial. At the end of the trial, if a
+   paid subscription has not been added to the database, it will no longer be accessible.*
+
+Issue a refund with a credit note
+=================================
+
+A *credit note* is a document issued to a customer informing them that they have been credited a
+certain amount of money. They can be used to provide a full refund to a customer, or to adjust any
+remaining amount due. While they are usually created through the :menuselection:`Accounting` or
+:menuselection:`Invoicing` applications, they can be created through a :menuselection:`Helpdesk`
+:guilabel:`Ticket`.
+
+.. note::
+   Invoices must be posted before a credit note can be generated.
+
+To create a credit note from a :menuselection:`Helpdesk` :guilabel:`Ticket`, click on the
+:guilabel:`Refund` button in the upper left corner. Select the corresponding invoice from the
+:guilabel:`Invoices to Refund` drop-down.
```

Comment:
This can be rewritten to justify the tags more clearly
- is **Ticket** the name of a heading, label, field, value...etc?
- upper left --> upper-left (hyphenated, since it's it's modifying a noun)
- drop-down --> drop-down menu (otherwise the hyphen doesn't make sense and "drop down" is incorrect)

```suggestion
Create a credit note by navigating into a ticket on the :menuselection:`Helpdesk` application, and
clicking the :guilabel:`Refund` button in the upper-left corner of the ticket dashboard. Then,
select the corresponding invoice from the :guilabel:`Invoices to Refund` drop-down menu.
```